BlueprintUICommonControls/Tests/Sources/BoxTests.swift

```diff
@@ -87,6 +87,16 @@ class BoxTests: XCTestCase {
             of: element,
             size: CGSize(width: 100, height: 100))
     }
+
+    func test_largeCornerRadius() {
```

Comment:
This test crashes on `main` on iOS 11

Reply:
The snapshot looks real funky... is that intentional?

Reply:
That's the "diamond" shape without clipping - but am going to update to use the min of width/height

Reply:
Ah right; looks way better now